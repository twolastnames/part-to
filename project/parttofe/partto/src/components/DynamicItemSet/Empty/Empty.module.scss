.empty {
  overflow-x: wrap;
  width: 100%;
}
