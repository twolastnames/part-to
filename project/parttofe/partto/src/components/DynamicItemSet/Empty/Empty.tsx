import React from "react";

import classes from "./Empty.module.scss";

export interface EmptyProps {
  content: React.ReactElement;
}

export function Empty({ content }: EmptyProps) {
  return (
    <div className={classes.empty} data-testid="Empty">
      {content}
    </div>
  );
}
