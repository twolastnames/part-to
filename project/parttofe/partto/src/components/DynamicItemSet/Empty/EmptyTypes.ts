export interface EmptyProps {
  content: React.ReactElement;
}
