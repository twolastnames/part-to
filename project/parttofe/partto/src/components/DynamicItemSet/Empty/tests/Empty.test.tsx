import React from "react";
import { describe, expect, test } from "@jest/globals";
import { render, screen } from "@testing-library/react";
import { Empty } from "../Empty";

describe("Empty", () => {
  test("snapshot", () => {
    render(<Empty content={<div>Empty</div>} />);
    const component = screen.getByTestId("Empty");
    expect(component).toMatchSnapshot();
  });
});
