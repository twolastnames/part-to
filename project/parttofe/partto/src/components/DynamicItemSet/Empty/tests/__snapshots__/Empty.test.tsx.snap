// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Empty snapshot 1`] = `
<div
  class="empty"
  data-testid="Empty"
>
  <div>
    Empty
  </div>
</div>
`;
